src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            int supplyTotal = 0;
+            int buyTotal = 0;
+            String line;
 
+            while ((line = reader.readLine()) != null) {
+                String[] fields = line.split(",");
+                String operationType = fields[0].trim();
+                int amount = Integer.parseInt(fields[1].trim());
```

Comment:
```suggestion
                String[] fields = line.split(DELIMITER);
                String operationType = fields[OPERATION_INDEX].trim();
                int amount = Integer.parseInt(fields[VALUE_INDEX].trim());
```
create constants please